.migration/docs/setup-guide.md

```diff
@@ -0,0 +1,204 @@
+# S2DM Migration Workflow Setup Guide
+
+This guide explains how to set up the automated S2DM migration workflow in your repository. The workflow automatically generates artifacts (GraphQL schemas, JSON schemas, registry files, and SKOS RDF) when changes are made to your specification files.
+
+## Prerequisites
+
+- A repository with S2DM specification files in a `spec/` directory
+- A `units.yaml` file at the repository root
```

Comment:
This is wrong. `S2DM` is the one that provides predefined commonly used elements. Among them, [the unit_enums.graphql file](https://github.com/COVESA/s2dm/blob/main/src/s2dm/spec/unit_enums.graphql).
For now, this file is given as a reference that mirrows the [units from vss](https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/units.yaml). However, there is already the plan to adopt and reuse standard model for units to construct this units file automatically (see #43).